app/views/DefaultBranchListPage.scala.html

```diff
@@ -17,18 +17,24 @@
 @import play.api.mvc.Call
 @import uk.gov.hmrc.cataloguefrontend.DateHelper._
 @import uk.gov.hmrc.cataloguefrontend.connector.RepositoryDisplayDetails
+@import uk.gov.hmrc.cataloguefrontend.service.DefaultBranchesService
 @import uk.gov.hmrc.cataloguefrontend.ViewMessages
 @import helper._
+@import scala.concurrent.Future
 
 @this(viewMessages: ViewMessages)
 
 @(repositories: Seq[RepositoryDisplayDetails],
-form: Form[_]
+    calculate: Int,
```

Comment:
Should this be named something more explicit, like `updatedDefaultBranchCount`?
You are already passing `repositories` in, I don't think we need `calculate` as well - It's simpler to understand `@{repositories.filter(_.defaultBranch != "master).count} / @{repositories.count}`